apps/hexis_init.py

```
@@ -1235,22 +1295,31 @@ def _post_init_handoff() -> int:
-            default=1,
-        ))
-    except (KeyboardInterrupt, Exception):
-        console.print("[muted]Run `hexis chat` to say hello.[/muted]")
+    while True:
+        try:
+            choice = asyncio.run(_prompt_choice(
+                "What now?",
+                ["Open chat now", "Open the web dashboard", "Exit"],
+                default=1,
+            ))
+        except (KeyboardInterrupt, Exception):
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)